File: blackjack/src/blackjack/Player.java
package blackjack;

import java.util.List;

/**
 * Created by Joost on 30-4-2017.
 */
public class Player {
    public List<Cards> Hand;
    public String Name;

    public Player(String name)
    {
        this.Name = name;
    }

    public void CheckHand()
    {
        int output = 0;
        for(int i = 0; i < Hand.size(); i++){
            output += Hand.get(i).value;
        }
        System.out.println("Uw totaal is " + output);
    }
}

File: Blackjack C#/Blackjack/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Blackjack
{
    class Player : IPlayer
    {
        public string Name { get; }
        public IDeck Hand { get; }

        public Player(string name)
        {
            this.Name = name;
            this.Hand = new Hand();
        }
        
        public int CheckHand()
        {
            int output = 0;

            for (int i = 0; i < this.Hand.cardList.Count; i++)
            {
                output += this.Hand.cardList[i].value;
            }

            return output;
        }

        
    }
}
